WorkloadManagementSystem/PilotAgent/pilotCommands.py

```diff
@@ -504,9 +504,13 @@ def execute( self ):
     self.cfg.append(
         '-o "/Resources/Computing/CEDefaults/NumberOfProcessors=%d"' % numberOfProcessors)
 
-    maxRAM = self.pp.queueParameters.get('MaxRAM', maxRAM)
+    maxRAM = int(self.pp.queueParameters.get('MaxRAM', maxRAM))
     if maxRAM:
-      self.cfg.append( '-o "/Resources/Computing/CEDefaults/MaxRAM=%d"' % maxRAM )
+      try:
+        self.cfg.append(
+            '-o "/Resources/Computing/CEDefaults/MaxRAM=%d"' % int(maxRAM))
+      except ValueError:
+        self.log.warn("MaxRAM is not an integer, won't filling it")
```

Comment:
"`won't be filling it`" or something without a quote for easier grepping... "`will not fill it`"...